Cubical/Algebra/Matrix.agda

```diff
@@ -169,14 +188,13 @@ module _ (G' : AbGroup ℓ) where
   -- VecMatrix that is equal to the one on FinMatrix
   VecMatrixAbGroup : (m n : ℕ) → AbGroup ℓ
   VecMatrixAbGroup m n =
-    InducedAbGroup (FinMatrixAbGroup G' m n) addVecMatrix
-      FinMatrix≃VecMatrix (FinMatrix→VecMatrixHomAdd m n)
+    InducedAbGroup (FinMatrixAbGroup G' m n) addVecMatrix zeroVecMatrix negVecMatrix FinMatrix≃VecMatrix
```

Comment:
@aljungstrom This is the fix to the Matrix code with the new InducedAbGroup. Can you fix the DirectProof code? 

Alternatively we can have InducedAbGroupFromAbGroupHom which only requires preservation of +, but this seems bad as we also want to be able to pick what the 0 and - of the induced group are...